DistFiles/reference_texts/Russian/MRK.xml

```diff
@@ -5166,7 +5166,7 @@
       <verse num="15" />
       <text>Shall we give, or shall we not give?»</text>
```

Comment:
Looks to me like this needs an opening quote mark

Reply:
No, in the English, the quote starts in the previous verse.

Reply:
Yikes, real life is never simple...